integration/test/utils/utils.go

```diff
@@ -388,3 +389,13 @@ func MarshalStack(err error) interface{} {
 	}
 	return string(trace)
 }
+
+func Execute(command string, args ...string) (out string, err error) {
```

Comment:
is this needed or can the similar function at line 47 be used? Maybe the one at 47 needs a timeout also? If both are needed, maybe move this one up beside the other one